README.md

```diff
@@ -129,147 +201,48 @@ m.collatz(arg_gen)
 for idx, val_list in enumerate(m.collect_returns()):
     print("State", idx, "::", val_list[0])
 ```
+</details>
 
 ## Requirements
+* Manticore requires Python 3.6 or greater 
+* Manticore officially supports the latest LTS version of Ubuntu
```

Comment:
Ubuntu 20.04 is technically the "latest LTS" version. Do we test on that now?

Reply:
Our CI runs against `ubuntu-latest`, so we'll start testing against it automatically as soon as it's available. For now, added a comment about GH Actions to clarify this.

Reply:
Perfect 👍